1-js-and-async-programming/1-javascript-fundamentals/3-factorial-chain.js

```diff
@@ -31,6 +31,19 @@ Requirements:
 
 */
 
-const factorialChain = (number, lastDigits) => {};
-
-module.exports = factorialChain;
+const factorialChain = (number, lastDigits) => {
+    if(number < 0 || lastDigits < 0) throw Error('Only possitive numbers are accept');
+    let factorialSum = 0;
+    let factorial = 1;
+    for(let i = 1; i <= number ; i++){
+        factorial = factorial * i;
+        factorialSum += acc;
```

Comment:
Did you mean `factorial`?
```suggestion
        factorialSum += factorial;
```